packages/web/src/pages/saved-page/hooks/useCollectionsData.tsx

```diff
@@ -1,87 +1,49 @@
 import { useMemo } from 'react'
 
```

Comment:
so what's up with this hook now?

Reply:
to merge in local collections? id say if we are comfortable about solving this using react-query correctly id say we can drop this complexity?

Reply:
I went down this path and it was pretty messy. Down to chat about designs for it on Monday though!